wiki/Community/Reddit/en.md

```diff
@@ -0,0 +1,40 @@
+# Reddit
+
+This article lists places on [Reddit](https://reddit.com) where users are able to interact with osu! and its community.
+
+## Gameplay
+
+These unofficial community-driven subreddits focus mainly on gameplay topics.
+
+| Avatar | Name | Description |
+| :-: | :-: | :-- |
+| ![osugame avatar](img/osugame.png) | [/r/osugame](https://reddit.com/r/osugame) | **/r/osugame** is currently the largest multi-mode forum open for general discussions and score showcases. |
+| *None* | [/r/osumania](https://reddit.com/r/osumania) | **/r/osumania** offers a safe space specifically intended for the [osu!mania](/wiki/Game_mode/osu!mania) game mode. |
+
+## Mapping
+
+These unofficial subreddits focus on providing help for newcomers as well as experienced mappers to dive into [mapping](/wiki/Beatmapping).
+
+| Avatar | Name | Description |
+| :-: | :-: | :-- |
+| *None* | [/r/osumapping](https://reddit.com/r/osumapping) | **/r/osumapping** serves as a platform all around mapping. |
+
+## Skinning
+
+These unofficial subreddits aim to provide resources surrounding [skinning](/wiki/Skinning), facilitating users to get into skinning more easily regardless of their experience. 
+
+| Avatar | Name | Description |
+| :-: | :-: | :-- |
+| ![osuskins avatar](img/osuskins.png) | [/r/osuskins](https://reddit.com/r/osuskins) | **/r/osuskins** allows users to ask for general skinning support, exchange design choices as well as request help for finding specific skins. |
+
+## Miscellaneous
+
+| Avatar | Name | Description |
+| :-: | :-: | :-- |
+| ![osureport avatar](img/osureport.png) | [/r/osureport](https://reddit.com/r/osureport) | **/r/osureport** is a hub designed for raising awareness about illegitimate players and cheated plays. It is not affiliated with the [osu! team](/wiki/People/osu!_team). |
```

Comment:
most official messaging (wiki, peppy, tournament committee etc.) about cheating/foul play on osu! is about not bringing attention to them. with that in mind i'm not sure about linking a hub expressly for accusing other players

Reply:
Not sure about this, so I'll defer judgement. But it definitely doesn't need the final sentence (we *do* watch over it, just like other reddits, to some extent).

Reply:
I think we should avoid including r/osureport here. It's certainly an osu related reddit with semiofficial blessing, but I'd like to avoid any confusing messaging on the website about reporting foul play. I think there should be one and only one way that osu.ppy.sh recommends reporting users (the report button).